MinecraftClient/ChatBots/ChatLog.cs

```diff
@@ -92,6 +100,14 @@ public override void GetText(string text)
             }
         }
 
+        public override void OnInternalCommand(string commandName,string commandParams, string Result)
```

Comment:
C# [naming conventions](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions) involve using `PascalCase` for method names and class attributes, and `camelCase` for local variables and arguments, so here this argument should be `string result`. You'll encounter some `incorrect_names` inside MCC's code, this is however not compliant and should be improved whenever possible. 😉